README.md

```diff
@@ -306,22 +306,31 @@ task.plugins {
 
 #### Default outputs
 
-**Java** projects: the `java` builtin is added by default.  If you
-wish to remove this output, for example, to only generate for Python:
+**Java** projects: the `java` builtin is added by default: without any further specification, Java classes will be generated in the `$generatedFilesDir/main/java` folder.
```

Comment:
The output path is not correct. Because the construction of the path is rather lengthy, it's probably not a good idea to repeat it in every section. Better just to refer to the "Change where the generated files are" section.

Reply:
👍